### DISCUSSION TITLE
Blazor

### DISCUSSION THREAD

**Discussion by TrailBlazor** [saadeghi/daisyui#801] [General]
Can you use the components for Blazor development?

Thanks,
Dave

**Comment by saadeghi** [comment]
For a non-Node environment, you can use the [CDN file](https://unpkg.com/daisyui@2.15.0/dist/full.css)  
And if you can purge unused CSS with a tool or plugin somehow, It's better to download the CDN file and purge unused class names so your final CSS file won't be large.

**Reply by TrailBlazor** [reply]
Thanks for the link to the CDN. Not sure if I want to use it for production purposes but worth doing to see if I even like Daisy.

**Reply by saadeghi** [reply]
Alright. I don't have any experience with Blazor but if you have any questions about daisyUI, let me know.

**Comment by AshSimmonds** [comment]
Don't need to use CDN, Trailering and Daisy are easy to implement. First
time can be tricky, but now I can spin up a clean Blazor TW/D environment
in a few minutes.

Just have to get used to remembering when a running instance needs a full
restart (injecting new/purging classes) or when a hot reload of the css
will work.

**Reply by TrailBlazor** [reply]
Thanks. Are there any docs or details on how to do that?

**Reply by TrailBlazor** [reply]
Okay, I figured out how to add it to my project.  Thanks.

**Reply by saadeghi** [reply]
Great!  
Sharing the solution might help others who see this conversation 🙏

**Reply by TrailBlazor** [reply]
Seems silly now but all I did was go to the install page: https://daisyui.com/docs/install/ and followed the directions.

**Comment by AshSimmonds** [comment]
*Trailering = Tailwind :/

**Comment by AshSimmonds** [comment]
This is the repo I originally learned from in January, and tweaked a few
minor steps to my own taste. Might be a couple changes since then, but it's
pretty straight-forward. Once you've done it a few times it becomes second
nature.

 - https://github.com/tesar-tech/BlazorAndTailwind
